community/socat/sources

```diff
@@ -0,0 +1 @@
+http://www.dest-unreach.org/socat/download/socat-1.7.3.3.tar.gz
```

Comment:
No `https` source available?

Reply:
Apparently they use a self-signed cert. We could also use https://repo.or.cz/socat.git/snapshot/e1bc39a53f808f4fdaeb3743c415da861789f351.tar.gz which is the snapshot of 1.7.3.3. What do you think?

Reply:
`http` is fine if no `https` is available as we also have the checksum verification to fallback on. :+1: